app/components/listing/OrganizationCard.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useMemo } from 'react';
+import { Link } from 'react-router-dom';
+import StreetViewImage from 'components/maps/StreetViewImage';
+import { Organization } from '../../models';
+
+export const OrganizationCard = ({ org }: { org: Organization }) => {
+  const address = org.addresses[0];
```

Comment:
Wait, is this the reason this has been broken the whole time? That the old code had `org.address`, which is not a real field? I can see in production that every time I try to hover over the link, it causes a Sentry error to fire with "Cannot read properties of undefined (reading 'address_1')".

Thank you for finally fixing this, now that we actually have type checking.